queries/query/highlights.scm

```diff
@@ -28,7 +28,7 @@
  (#match? @number "^[-+]?[0-9]+(.[0-9]+)?$"))
```

Comment:
would be nice to convert this as well, but I don't think Lua patterns support optional groups.

Reply:
yeah it'd be two queries then